deps/wazuh_testing/wazuh_testing/tools/performance/visualization.py

```diff
@@ -126,6 +127,23 @@ def _set_x_ticks_interval(self, ax):
             ax.xaxis.set_major_locator(mdates.MinuteLocator(interval=self.x_ticks_interval))
         ax.xaxis.set_major_formatter(mdates.DateFormatter('%H:%M:%S'))
 
+    @staticmethod
+    def _get_statistics(df, calculate_mean=True, calculate_median=False):
+        """Function for calculating statistics.
+
+        Args:
+            df (pandas.DataFrame): dataframe on which the operations will be applied.
+            calculate_mean (bool, optional): specify whether or not the mean will be calculated.
+            calculate_median (bool, optional): specify whether or not the median will be calculated.
+        """
+        statistics = str()
+        if calculate_mean:
+            statistics += f"Mean: {round(mean(df), 3)}\n"
+        if calculate_median:
+            statistics += f"Median: {round(median(df), 3)}\n"
```

Comment:
I recommend you using the Pandas method to compute the mean and the median. 
- [mean](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.mean.html#pandas-dataframe-mean)
- [median](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.median.html)
Both of them are way faster than regular methods from `statistics`.

Reply:
Sounds good. Done!